ionic/platform/storage/storage.ts

```diff
@@ -13,13 +13,13 @@
 export class Storage {
   private _strategy: any;
 
-  constructor(strategyCls: any, options) {
+  constructor(strategyCls: IStorageEngine, options: any) {
```

Comment:
same here, options is an optional param. Please consider changing it to options?, this way the signature will match.